server/publicapi/features/prepopulate.feature

```diff
@@ -7,7 +7,7 @@ Feature: Prepopulate public content
     	"""
         Then we get response code 201
 
-        When we get "/items"
+        When we get "/items?start_date=1970-01-01"
```

Comment:
How can perform a client a pooling for new items added on current day?
If we use this day granularity function we will get at every request all items from the current day.
We have to use a datetime or better a change counter.